docs/rules/order-in-components.md

```diff
@@ -64,24 +64,26 @@ export default {
 If you want you can change the order providing the optional configuration in your `.eslintrc` file. Setting responsible for the above order looks like this:
 
 ```
-vue/order-in-components: [2, {
-  order: [
-    ['name', 'delimiters', 'functional', 'model'],
-    ['components', 'directives', 'filters'],
-    ['parent', 'mixins', 'extends', 'provide', 'inject'],
-    'el',
-    'template',
-    'props',
-    'propsData',
-    'data',
-    'computed',
-    'watch',
-    'lifecycle_hooks',
-    'methods',
-    'render',
-    'renderError'
-  ]
-}]
+{
+  'vue/order-in-components': [2, {
+    order: [
+      ['name', 'delimiters', 'functional', 'model'],
+      ['components', 'directives', 'filters'],
+      ['parent', 'mixins', 'extends', 'provide', 'inject'],
+      'el',
+      'template',
+      'props',
+      'propsData',
+      'data',
+      'computed',
+      'watch',
+      'LIFECYCLE_HOOKS',
+      'methods',
+      'render',
+      'renderError'
+    ]
+  }
+]
```

Comment:
I think this is actually wrong. Current version seems fine @armano2, why do you want to change it?